services/IoT/applications/libraries/nodejs.md

```diff
@@ -245,7 +245,7 @@ By default, applications subscribe to all events from all connected devices. Use
 	});
 ```
 
-### Subscribing to all events published by a device in JSON format
+### Subscribing to all events that are published in JSON format
```

Comment:
the phrase "by a device" is implied by "published" - the conceptual info has already made clear that devices do the publishing. The phrase was making "in JSON format" dangle at the end of the sentence, which made it seem like the device is in JSON format.